app/experimenter/nimbus-ui/src/App.tsx

```diff
@@ -0,0 +1,17 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from "react";
```

Comment:
Nit: would love if we could prefer single quotes like FxA

Reply:
This comes from the root-level `.prettierrc` and the rest of the project uses double quotes. I prefer single quotes as well but unless we want to change it project-wide I think we might need to stick with double for this project (or add a separate prettier file in `nimbus-ui`).

Reply:
Lame but understandable. Carry on!